### ISSUE TITLE
ascii decoding for database response

### ISSUE THREAD

**Issue by Sunno** [chtd/psycopg2cffi#50]
Hi! I'm having an issue, in method "bytes_to_ascii" in _impl/adapters.py it throws an UnicodeDecodeError since database response has special characters (it's not in english), for instance in my error is:
```
 'ERROR:  llave duplicada viola restricción de unicidad «companies_requiredfield_branch_id_7e08660e8fafa825_uniq»
DETAIL:  Ya existe la llave (branch_id, field_id)=(2, 60).'
```
This database error is actually an integrity error (because an unique constraint), in my app (written in django) I capture it and send a message to user, but since I get an UnicodeDecodeError instead of an IntegrityError, for now I'll just validate it before trying to save.

I tried changing this line in _impl/adapters.py
```python
return b.decode('ascii')
```
By
```python
return b.decode('utf-8')
```
and it worked

**Comment by lopuhin** [comment]
@Sunno I think this is the same issue as a just recently fixed #56

**Comment by lopuhin** [comment]
2.7.3 release is on PyPI
